client/src/components/Utilities/GetCoordinates/GetCoordinates.tsx

```diff
@@ -356,19 +357,39 @@ const GetCoordinates: React.FC = () => {
             </Box>
             {genomicCoordinateInfo}
             <Box className={classes.fieldsPair}>
-              <TextField
-                margin="dense"
-                label="Genomic Start"
-                value={start}
-                onChange={(event) => setStart(event.target.value)}
-                helperText={start ? startText : ""}
-              />
-              <TextField
-                margin="dense"
-                label="Genomic End"
-                value={end}
-                onChange={(event) => setEnd(event.target.value)}
-              />
+              <HelpTooltip
+                placement="bottom"
+                title={
+                  <Typography>
+                    The starting genomic position of the transcript segment.
+                    Inter-residue.
+                  </Typography>
+                }
+              >
+                <TextField
+                  margin="dense"
+                  label="Genomic Start"
+                  value={start}
+                  onChange={(event) => setStart(event.target.value)}
+                  helperText={start ? startText : ""}
+                />
+              </HelpTooltip>
+              <HelpTooltip
+                placement="bottom"
+                title={
+                  <Typography>
+                    The ending genomic position of the transcript segment.
+                    Inter-residue
```

Comment:
Be consistent with period
```suggestion
                    Inter-residue.
```